server/src/main/scala/com/cloudera/livy/LivyConf.scala

```diff
@@ -166,6 +170,37 @@ object LivyConf {
     "spark.yarn.jars"
   )
 
+  // TODO: Add Conf Deprecation
+  case class DepConf(override val key: String,
+                     override val version: String,
```

Comment:
Thanks, I found an example of how this indentation is supposed to work, we really need to clean up of style checker, it seems it misses a lot.